spinalcordtoolbox/reports/slice.py

```diff
@@ -347,7 +378,10 @@ def get_dim(self, image):
     def get_center(self, img_idx=-1):
         """Get the center of mass of each slice. By default, it assumes that self._images is a list of images, and the
         last item is the segmentation from which the center of mass is computed."""
-        image = self._images[img_idx]
+        if self._image_seg is None:
+            image = self._images[img_idx]
+        else:
+            image = self._image_seg
```

Comment:
It might be good to update the help description to reference the new behavior of using the segmented spinal cord by default if it's present.